<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>659</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Оповеститель</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding: 10px;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Фамилия</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="surname">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Иванов</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Имя</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="name">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Иван</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Отчество</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="patronymic">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Иванович</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Почта</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="mail">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>mail@gmail.com</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>10</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">display: flex;
justify-content: flex-center;</string>
              </property>
              <property name="text">
               <string>или</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Номер телефона</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="phone">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>+79999999999</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Дата и время приема</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QDateTimeEdit" name="date">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="11" column="0" colspan="2">
             <widget class="QPushButton" name="send_btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Добавить на отправку</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="2">
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="12" column="0" colspan="2">
             <widget class="QPushButton" name="table_btn">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Посмотреть данные на отправку</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin: 10px;</string>
            </property>
            <property name="text">
             <string>Данные на отправку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="picture">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Настройки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Адрес</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="set_email">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhEmailCharactersOnly</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="set_password">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhNone</set>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Почта отправителя</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Номер</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="set_phone">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Телефон</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="label_21">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Тема</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="set_topic">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_22">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Напоминание за неделю</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="set_notification">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_23">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Напоминание о начале</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="set_reception_began">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_20">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Сообщения и письма</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QLabel" name="label_24">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <italic>false</italic>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Примечание: вместо даты нужно ставить &quot;{}&quot;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="save_settings_btn">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
